meta-lmp-bsp/dynamic-layers/meta-ti-bsp/recipes-bsp/u-boot/u-boot-ti-staging/lib-zlib-Fix-a-bug-when-getting-a-gzip-header-extra-field.patch

```diff
@@ -0,0 +1,150 @@
+ kDk3Ipkvgx2P1ePSqZxE3bsQYH0sZMGsXx69Sayap75DgWpWfG6JNHLykqf6CR1/J5AF
+ bVnQ==
+X-Google-DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed;
+ d=1e100.net; s=20221208; t=1686840887; x=1689432887;
+ h=content-transfer-encoding:mime-version:message-id:date:subject:cc
+ :to:from:x-gm-message-state:from:to:cc:subject:date:message-id
+ :reply-to;
+ bh=ZlcZ/TvSHRdVMgcEyiYYvZoSkkmQihHQknCJfDU7dXU=;
+ b=ERbykPOQJDOHGQhnfs26hT8AtIzb/1J9Y0AzyoM+yxKxnofc4kOq16HB98yzSbbv8B
+ g2QXsfxHCysvG9SgIG9hQSsfhG/4GW4ATwZrgGx3C0BAe8l7K4FhrK9IRfQWgztMC9QN
+ 8cUbabeNzQaa9w1shRIuQaJrthkg9/GqE2GdaassqtTfPnM+AUrUrEFVwG6sgjs5zwZw
+ 7rfBz3/k+4hbB8UjqQsM77YfwqXG495cN5ENfPrdeC6YWEIeMVhop4knbzdoCLAfSjbd
+ 0uT0WPxrrWtcM99FZuqaFVUtFjhP1yauIjkZRjLrd5rHiWCj2e7ns/pA0OASS3P+Q8Co
+ nTMw==
+X-Gm-Message-State: AC+VfDyPc9oFK6n9Mzw6zdmejxbO2SeZJtBMOZ9f5XxzZfEFUBIOilO7
+ Dum6u/M5ISm22SW9judHyAEgDP/Q9h0yajN/Qt8fvw==
+X-Google-Smtp-Source: 
+ ACHHUZ5Y3bCKlS9IjeXX4/fZEmVixhD/VdCyBxetYirrE2WP+nIselSwtBGxaL8AwojYk7/8Z1K8AQ==
+X-Received: by 2002:adf:db03:0:b0:311:17c5:3a9a with SMTP id
+ s3-20020adfdb03000000b0031117c53a9amr1286227wri.38.1686840886786;
+ Thu, 15 Jun 2023 07:54:46 -0700 (PDT)
+Received: from localhost.localdomain ([194.104.22.162])
+ by smtp.gmail.com with ESMTPSA id
+ b1-20020a5d5501000000b0030ae69920c9sm21323856wrv.53.2023.06.15.07.54.45
+ (version=TLS1_3 cipher=TLS_AES_256_GCM_SHA384 bits=256/256);
+ Thu, 15 Jun 2023 07:54:46 -0700 (PDT)
+From: Oleksandr Suvorov <oleksandr.suvorov@foundries.io>
+To: u-boot@lists.denx.de
+Cc: Oleksandr Suvorov <oleksandr.suvorov@foundries.io>
+Subject: [PATCH] lib/zlib: Fix a bug when getting a gzip header extra field
+Date: Thu, 15 Jun 2023 17:54:34 +0300
+Message-Id: <20230615145434.103140-1-oleksandr.suvorov@foundries.io>
+X-Mailer: git-send-email 2.40.1
+MIME-Version: 1.0
+X-BeenThere: u-boot@lists.denx.de
+X-Mailman-Version: 2.1.39
+Precedence: list
+List-Id: U-Boot discussion <u-boot.lists.denx.de>
+List-Unsubscribe: <https://lists.denx.de/options/u-boot>,
+ <mailto:u-boot-request@lists.denx.de?subject=unsubscribe>
+List-Archive: <https://lists.denx.de/pipermail/u-boot/>
+List-Post: <mailto:u-boot@lists.denx.de>
+List-Help: <mailto:u-boot-request@lists.denx.de?subject=help>
+List-Subscribe: <https://lists.denx.de/listinfo/u-boot>,
+ <mailto:u-boot-request@lists.denx.de?subject=subscribe>
+Errors-To: u-boot-bounces@lists.denx.de
+Sender: "U-Boot" <u-boot-bounces@lists.denx.de>
+X-Virus-Scanned: clamav-milter 0.103.8 at phobos.denx.de
+X-Virus-Status: Clean
+
+This fixes CVE-2022-37434 [1] and bases on 2 commits from Mark
+Adler's zlib master repo - the original fix of CVE bug [2] and
+the fix for the fix [3].
+
+[1]
+https://github.com/advisories/GHSA-cfmr-vrgj-vqwv
+[2]
+https://github.com/madler/zlib/commit/eff308af425b67093bab25f80f1ae950166bece1
+[3]
+https://github.com/madler/zlib/commit/1eb7682f845ac9e9bf9ae35bbfb3bad5dacbd91d
+
+Fixes: e89516f031d ("zlib: split up to match original source tree")
```

Comment:
Can you please add the upstream status?
```
Upstream-Status: Submitted [url]
```

Reply:
Done.

Also I added CVE field so that cve-check automatically adds this CVE to SBOM as fixed in this u-boot.